File: cms-adapter/src/Rsbc.Dmf.CaseManagement/BringForwardRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Rsbc.Dmf.CaseManagement
{
    public class BringForwardRequest
    {
        public string CaseId { get; set; }
        public string Assignee { get; set; }
        public string Subject { get; set; }
        public string Description { get; set; }
        public BringForwardPriority? Priority { get; set; }

    }
}

File: driver-portal/src/ClientApp/src/app/shared/api/models/bring-forward-request.ts
/* tslint:disable */
/* eslint-disable */
import { BringForwardPriority } from '../models/bring-forward-priority';
export interface BringForwardRequest {
  assignee?: string | null;
  caseId?: string | null;
  description?: string | null;
  priority?: BringForwardPriority;
  subject?: string | null;
}
